pdns/lua-base4.cc

```diff
@@ -36,6 +37,31 @@ void BaseLua4::loadString(const std::string &script) {
   loadStream(iss);
 };
 
+int BaseLua4::includePath(const std::string& path) {
```

Comment:
why `int`? the only return statement in here says `0`

Reply:
meh, i suppose this should fail if loading any of the files fails.